src/test-apps/happy/lib/WeaveDeviceManager.py

```diff
@@ -558,6 +595,7 @@ def RunWdmClientTest():
     testWdmClientDataSinkSetFlushRefreshGetData(testObject)
     testWdmClientDataSinkRefreshGetDataRefresh(testObject)
     testWdmClientDataSinkRefreshIndividualGetDataRefresh(testObject)
+    testWdmClientDataSinkRefreshDataWithEvents(testObject)
```

Comment:
Please fix travis test failure,
do you need to add any modification for mock-device for events generation?
could you add another test for large events, which can trigger multiple notificaation.

Reply:
> Please fix travis test failure,
> do you need to add any modification for mock-device for events generation?
> could you add another test for large events, which can trigger multiple notificaation.

I have no idea about the two failing tests, the failed test is `happy/tests/standalone/pairing/test_weave_pairing_01.py`

Reply:
testWdmClientDataSinkRefreshDataWithEvents is part for test_weave_pairing_01.py.

Reply:
you should be able to run it locally via running  test_weave_pairing_01.py